tools/codegen/config/config_model.go

```diff
@@ -9,7 +9,6 @@ type Resource struct {
 	Read          *APIOperation `yaml:"read"`
 	Update        *APIOperation `yaml:"update"`
 	Delete        *APIOperation `yaml:"delete"`
-	Wait          *WaitConfig   `yaml:"wait"`
```

Comment:
Thanks for this clean up, makes it clear up to what point (template model) hardcoded values are defined.